docs/source/models_new.rst

```diff
@@ -191,6 +357,38 @@ pre-trained weights:
 
    models/video_resnet
 
+|
+
+Here is an example of how to use the pre-trained video classification models:
+
+.. code:: python
+
+
+    from torchvision.io.video import read_video
+    from torchvision.models.video import r3d_18, R3D_18_Weights
+
+    vid, _, _ = read_video("test/assets/videos/v_SoccerJuggling_g23_c01.avi")
```

Comment:
@bjuncek Apparently this emits a vague deprecation warning. Does it make sense to remove the warning given that we are yet uncertain what we will do with the specific API? Or perhaps I should modify my example to avoid the warnings?

Reply:
Yeah, feel free to remove it.
I'm in the process of re-writing the video-module to have a more sensical stucture

Reply:
See #6056